### DISCUSSION TITLE
Taro兼容微信WMPF车机小程序吗

### DISCUSSION THREAD

**Discussion by victorWuxz** [NervJS/taro#13205] [Q & A]
Taro兼容微信WMPF车机小程序吗

目前在做技术选型，要做车机微信小程序，支持直接在车机桌面展示，微信的WMPF支持，原生框架支持，请问uni-app对WMPF支持情况如何

**Comment by victorWuxz** [comment]
Taro兼容微信WMPF车机小程序吗

目前在做技术选型，要做车机微信小程序，支持直接在车机桌面展示，微信的WMPF支持，原生框架支持，请问Taro对WMPF支持情况如何

**Comment by AdvancedCat** [comment]
又是微信创造的新玩具，『真 · 学不动 』了。

目前我们还是主要做面向微信框架的小程序适配，我理解这个硬件框架应该也是移植的小程序框架到其他设备中运行，对于小程序开发者相对透明。当然也需要对 `wmpf.xxx` 相关API的适配。适合做成 `@tarojs/plugin-paltform-wmpf` 插件形式，以实现跨端适配。

如果有结合业务且专门面向 WMPF 小程序的开发者来牵头是最好的了，这样才有时间和精力去维护这样一个跨端插件，以确保质量和稳定性。
